docs/operators/remove.md

```diff
@@ -0,0 +1,343 @@
+| `id`       | `restructure`    | A unique identifier for the operator                                                                                                                                                                                                     |
+| `output`   | Next in pipeline | The connected operator(s) that will receive all outbound entries                                                                                                                                                                         |
+| `field`      | required       | The [field](/docs/types/field.md) to remove.
+| `on_error` | `send`           | The behavior of the operator if it encounters an error. See [on_error](/docs/types/on_error.md)                                                                                                                                          |
+| `if`       |                  | An [expression](/docs/types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
+
+Example usage:
+
+<hr>
+
+Remove value from body
+```yaml
+- type: remove 
+    field: 
+    - key1
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "key1": "val1",
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove object from body
+```yaml
+- type: remove 
+    field: 
+    - object
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }
+  },
+  "key": "val"
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { 
+     "key": "val"
+  }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove multiple Values from body
+```yaml
+- type: remove 
+    field: 
+    - object
+    - key
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    },
+    "key": "val"
+  },
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { },  
+  "body": { }
+}
+```
+
+</td>
+</tr>
+</table>
+
+<hr>
+
+Remove Value from attributes
+```yaml
+- type: remove 
+    field: 
+    - $attributes.otherkey
+```
+
+<table>
+<tr><td> Input Entry </td> <td> Output Entry </td></tr>
+<tr>
+<td>
+
+```json
+{
+  "resource": { },
+  "attributes": { 
+    "otherkey": "val"
+  },  
+  "body": {
+    "object": {
+      "nestedkey": "nestedval"
+    }, 
+    "key": "val"
+  },
```

Comment:
Since the example is focused on attributes, let's keep the body very simple. I think this body distracts from the point of the example.